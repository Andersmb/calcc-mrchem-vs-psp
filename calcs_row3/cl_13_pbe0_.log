Starting job 5688110 on c1-[21,26],c2-7,c3-[6,13],c5-7 at Thu May 19 20:22:56 CEST 2022

Copying cl_13_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/singlets
Copying cl_13_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/singlets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688110      cl_13_pbe+        180                                             01:45:27      0:0 
5688110.bat+      batch         24        1   00:00:04          0   00:00:04   01:45:27      0:0 
5688110.ext+     extern        180        6   00:00:00          5   00:00:00   01:45:28      0:0 
5688110.0      mrchem.x        180       30   01:44:23         29   05:06:32   01:44:34      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688110                                                                          
5688110.bat+     19336K          0     19336K        0              0          0 
5688110.ext+          0          5          0        0              5          0 
5688110.0     24106168K         29 8621647052        0             20          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688110                                                                                               
5688110.bat+       19.42M               0         19.42M        0.07M                0          0.07M 
5688110.ext+        0.00M               5          0.00M            0                5              0 
5688110.0           6.75M              12          6.73M      176.77M               11        126.73M 

Job 5688110 completed at Thu May 19 22:08:24 CEST 2022
