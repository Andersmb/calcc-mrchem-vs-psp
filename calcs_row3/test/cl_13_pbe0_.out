                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Cl            :        0.000000        0.066623       -0.027525
    1       O            :        0.000000       -2.551781       -0.560239
    2       O            :        0.000000        0.066623        3.073515
    3       O            :        2.609523        1.110319       -1.338050
    4       O            :       -2.609523        1.110319       -1.338050
    5       H            :        0.000000        1.855438        3.552749
---------------------------------------------------------------------------
 Center of mass          :        0.000000       -0.000000        0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              25
 Beta electrons          :                                              25
 Total electrons         :                                              50
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999999999013e-01
    1    1    a          :                              1.000000008888e+00
    2    1    a          :                              1.000000000875e+00
    3    1    a          :                              1.000000190707e+00
    4    1    a          :                              9.999999710771e-01
    5    1    a          :                              9.999999939423e-01
    6    1    a          :                              1.000000003007e+00
    7    1    a          :                              1.000000001890e+00
    8    1    a          :                              1.000000000195e+00
    9    1    a          :                              9.999999746011e-01
   10    1    a          :                              1.000000016343e+00
   11    1    a          :                              1.000000023429e+00
   12    1    a          :                              1.000000030753e+00
   13    1    a          :                              1.000000198564e+00
   14    1    a          :                              9.999999117698e-01
   15    1    a          :                              1.000000119829e+00
   16    1    a          :                              9.999989837185e-01
   17    1    a          :                              9.999997490986e-01
   18    1    a          :                              9.999999423629e-01
   19    1    a          :                              9.999994934982e-01
   20    1    a          :                              9.999995553176e-01
   21    1    a          :                              1.000000054371e+00
   22    1    a          :                              1.000000001844e+00
   23    1    a          :                              9.999994692926e-01
   24    1    a          :                              1.000000137296e+00
   25    1    b          :                              9.999999997444e-01
   26    1    b          :                              1.000000024477e+00
   27    1    b          :                              1.000000000876e+00
   28    1    b          :                              1.000000202832e+00
   29    1    b          :                              9.999999765024e-01
   30    1    b          :                              9.999999962793e-01
   31    1    b          :                              1.000000003224e+00
   32    1    b          :                              1.000000001944e+00
   33    1    b          :                              1.000000000177e+00
   34    1    b          :                              9.999999824808e-01
   35    1    b          :                              1.000000031006e+00
   36    1    b          :                              1.000000023318e+00
   37    1    b          :                              1.000000030741e+00
   38    1    b          :                              1.000000202055e+00
   39    1    b          :                              9.999999781780e-01
   40    1    b          :                              1.000000119895e+00
   41    1    b          :                              1.000000030107e+00
   42    1    b          :                              9.999997488951e-01
   43    1    b          :                              9.999997547867e-01
   44    1    b          :                              9.999983223487e-01
   45    1    b          :                              9.999995580869e-01
   46    1    b          :                              1.000000058746e+00
   47    1    b          :                              9.999999556538e-01
   48    1    b          :                              1.000000230155e+00
   49    1    b          :                              1.000000133062e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                752.937585144883
 E-N energy              :            (au)              -2333.065649110744
 Coulomb energy          :            (au)                612.198969979463
 Exchange energy         :            (au)                -15.200275483827
 X-C energy              :            (au)                -47.338286967063
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                274.292912412548
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)              -1030.467656437288
 Nuclear energy          :            (au)                274.292912412548
---------------------------------------------------------------------------
 Total energy            :            (au)             -7.561747440247e+02
                         :      (kcal/mol)             -4.745068120942e+05
                         :        (kJ/mol)             -1.985336503091e+06
                         :            (eV)             -2.057656212474e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)               -102.222560007115
    1    1    a          :            (au)                -19.290541736081
    2    1    a          :            (au)                -18.873952942979
    3    1    a          :            (au)                -18.862130102206
    4    1    a          :            (au)                -18.822119412071
    5    1    a          :            (au)                -10.511227636858
    6    1    a          :            (au)                 -8.265800986054
    7    1    a          :            (au)                 -8.262254652169
    8    1    a          :            (au)                 -8.261891973619
    9    1    a          :            (au)                 -1.573269651908
   10    1    a          :            (au)                 -1.296921372542
   11    1    a          :            (au)                 -1.139044475441
   12    1    a          :            (au)                 -1.075802371929
   13    1    a          :            (au)                 -0.895326554459
   14    1    a          :            (au)                 -0.668837066075
   15    1    a          :            (au)                 -0.607250440810
   16    1    a          :            (au)                 -0.573166509152
   17    1    a          :            (au)                 -0.455720801877
   18    1    a          :            (au)                 -0.441933540712
   19    1    a          :            (au)                 -0.328893036929
   20    1    a          :            (au)                 -0.263139839424
   21    1    a          :            (au)                 -0.256399258836
   22    1    a          :            (au)                 -0.227860938714
   23    1    a          :            (au)                 -0.224625683181
   24    1    a          :            (au)                 -0.198131765843
   25    1    b          :            (au)               -102.222560886084
   26    1    b          :            (au)                -19.290542751192
   27    1    b          :            (au)                -18.873953068987
   28    1    b          :            (au)                -18.862130790658
   29    1    b          :            (au)                -18.822122762766
   30    1    b          :            (au)                -10.511227504964
   31    1    b          :            (au)                 -8.265800514999
   32    1    b          :            (au)                 -8.262254068799
   33    1    b          :            (au)                 -8.261892304653
   34    1    b          :            (au)                 -1.573265728471
   35    1    b          :            (au)                 -1.296927296110
   36    1    b          :            (au)                 -1.139050139345
   37    1    b          :            (au)                 -1.075820250040
   38    1    b          :            (au)                 -0.895328239559
   39    1    b          :            (au)                 -0.668837180320
   40    1    b          :            (au)                 -0.607264320730
   41    1    b          :            (au)                 -0.573168806917
   42    1    b          :            (au)                 -0.455749213323
   43    1    b          :            (au)                 -0.441933748825
   44    1    b          :            (au)                 -0.328887975819
   45    1    b          :            (au)                 -0.263119505954
   46    1    b          :            (au)                 -0.256397625547
   47    1    b          :            (au)                 -0.227852315949
   48    1    b          :            (au)                 -0.224618594154
   49    1    b          :            (au)                 -0.198135065353
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -447.197643416501
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          7.071068e+00        -756.174744024739         -7.561747e+02
    1          6.263174e-01        -760.692307971461         -4.517564e+00
    2          2.516877e-01        -760.927846212790         -2.355382e-01
    3          1.415330e-01        -760.985477809852         -5.763160e-02
    4          7.671570e-02        -761.006930932015         -2.145312e-02
    5          1.065751e-01        -761.033065700683         -2.613477e-02
    6          2.985993e-02        -761.036463480640         -3.397780e-03
    7          3.657007e-02        -761.039342714608         -2.879234e-03
    8          2.100039e-02        -761.040040808863         -6.980943e-04
    9          3.818828e-02        -761.040837089969         -7.962811e-04
   10          1.481457e-02        -761.041271927685         -4.348377e-04
   11          1.640641e-02        -761.041567793912         -2.958662e-04
   12          1.035278e-02        -761.041692343991         -1.245501e-04
   13          8.727578e-03        -761.041775079507         -8.273552e-05
   14          6.044659e-03        -761.041829685394         -5.460589e-05
   15          5.724345e-03        -761.041864929255         -3.524386e-05
   16          4.087434e-03        -761.041881334772         -1.640552e-05
   17          4.513071e-03        -761.041896781884         -1.544711e-05
   18          2.822020e-03        -761.041903507805         -6.725921e-06
   19          2.031865e-03        -761.041907564744         -4.056939e-06
   20          1.709720e-03        -761.041910073710         -2.508965e-06
   21          9.456727e-04        -761.041911137507         -1.063797e-06
   22          7.282227e-04        -761.041911868830         -7.313226e-07
   23          5.620176e-04        -761.041912233252         -3.644227e-07
   24          5.084133e-04        -761.041912518344         -2.850920e-07
   25          3.726482e-04        -761.041912746245         -2.279003e-07
   26          2.241540e-04        -761.041912849301         -1.030560e-07
   27          1.970730e-04        -761.041912912343         -6.304299e-08
   28          1.794960e-04        -761.041912948956         -3.661239e-08
---------------------------------------------------------------------------
                      SCF converged in 28 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Cl            :        0.000000        0.066623       -0.027525
    1       O            :        0.000000       -2.551781       -0.560239
    2       O            :        0.000000        0.066623        3.073515
    3       O            :        2.609523        1.110319       -1.338050
    4       O            :       -2.609523        1.110319       -1.338050
    5       H            :        0.000000        1.855438        3.552749
---------------------------------------------------------------------------
 Center of mass          :        0.000000       -0.000000        0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                758.720768419450
 E-N energy              :            (au)              -2350.679977181210
 Coulomb energy          :            (au)                618.921135582635
 Exchange energy         :            (au)                -15.106527760383
 X-C energy              :            (au)                -47.190224421996
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                274.292912412548
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)              -1035.334825361504
 Nuclear energy          :            (au)                274.292912412548
---------------------------------------------------------------------------
 Total energy            :            (au)             -7.610419129490e+02
                         :      (kcal/mol)             -4.775610066813e+05
                         :        (kJ/mol)             -1.998115253252e+06
                         :            (eV)             -2.070900453244e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)               -102.085045822402
    1    1    a          :            (au)                -19.366149545363
    2    1    a          :            (au)                -19.286796784889
    3    1    a          :            (au)                -19.286650884651
    4    1    a          :            (au)                -19.286643562998
    5    1    a          :            (au)                 -9.862709420539
    6    1    a          :            (au)                 -7.598085422061
    7    1    a          :            (au)                 -7.594877716021
    8    1    a          :            (au)                 -7.594806440362
    9    1    a          :            (au)                 -1.318380722749
   10    1    a          :            (au)                 -1.166556905153
   11    1    a          :            (au)                 -1.071574277701
   12    1    a          :            (au)                 -1.041599145861
   13    1    a          :            (au)                 -0.810276115893
   14    1    a          :            (au)                 -0.675937923904
   15    1    a          :            (au)                 -0.608506860523
   16    1    a          :            (au)                 -0.598719093251
   17    1    a          :            (au)                 -0.542099487051
   18    1    a          :            (au)                 -0.469227312113
   19    1    a          :            (au)                 -0.441471868523
   20    1    a          :            (au)                 -0.422002802406
   21    1    a          :            (au)                 -0.398361502674
   22    1    a          :            (au)                 -0.370777244824
   23    1    a          :            (au)                 -0.360487462046
   24    1    a          :            (au)                 -0.340117556180
   25    1    b          :            (au)               -102.085045823947
   26    1    b          :            (au)                -19.366149544024
   27    1    b          :            (au)                -19.286796786904
   28    1    b          :            (au)                -19.286650886078
   29    1    b          :            (au)                -19.286643564413
   30    1    b          :            (au)                 -9.862709419899
   31    1    b          :            (au)                 -7.598085422058
   32    1    b          :            (au)                 -7.594877714609
   33    1    b          :            (au)                 -7.594806439310
   34    1    b          :            (au)                 -1.318380721106
   35    1    b          :            (au)                 -1.166556896431
   36    1    b          :            (au)                 -1.071574279779
   37    1    b          :            (au)                 -1.041599149367
   38    1    b          :            (au)                 -0.810276108486
   39    1    b          :            (au)                 -0.675937916170
   40    1    b          :            (au)                 -0.608506859869
   41    1    b          :            (au)                 -0.598719093306
   42    1    b          :            (au)                 -0.542099478176
   43    1    b          :            (au)                 -0.469227309484
   44    1    b          :            (au)                 -0.441471869648
   45    1    b          :            (au)                 -0.422002814383
   46    1    b          :            (au)                 -0.398361502989
   47    1    b          :            (au)                 -0.370777250075
   48    1    b          :            (au)                 -0.360487467057
   49    1    b          :            (au)                 -0.340117563535
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -445.195723761243
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.000000       -0.284853       -0.448777
 Magnitude               :            (au)                        0.531547
                         :         (Debye)                        1.351058
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.871882        1.782215
 Magnitude               :            (au)                        1.984053
                         :         (Debye)                        5.042959
---------------------------------------------------------------------------
 Total vector            :       -0.000000        0.587029        1.333437
 Magnitude               :            (au)                        1.456934
                         :         (Debye)                        3.703157
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  1h 44m 29s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
